internal/service/vpc/resource_gcp_vpc_peering_connection.go

```diff
@@ -241,5 +247,27 @@ func copyGCPVPCPeeringConnectionPropertiesFromAPIResponseToTerraform(
 		return diag.FromErr(err)
 	}
 
+	var toProjectId string
+	var toVPCNetwork string
+
+	si := *peeringConnection.StateInfo
+	if len(si) > 0 {
+		if v, ok := si["to_project_id"]; ok {
+			toProjectId = v.(string)
+		}
+		if v, ok := si["to_vpc_network"]; ok {
+			toVPCNetwork = v.(string)
+		}
+
+		if toProjectId != "" && toVPCNetwork != "" {
+			const gcpAPI = "https://www.googleapis.com/compute/v1"
```

Comment:
could this be moved to the top of the file?

Reply:
Why? We are not using it anywhere else except this function?

Reply:
constants are generally defined at the top of the file in Golang